aptos-move/framework/aptos-stdlib/sources/fixed_point64.move

```diff
@@ -112,6 +112,7 @@ module aptos_std::fixed_point64 {
     }
     spec create_from_rational {
         pragma opaque;
+        pragma timeout = 600; // Set larger timeout to avoid prover failure
```

Comment:
Please do not add such gigantic timeout. 600s is unacceptable. This verification need to be turned off because it is not working.